src/create_disk.sh

```diff
@@ -8,27 +8,61 @@
 # although see also https://github.com/coreos/coreos-assembler/pull/298
 # For people building "derived"/custom FCOS-like systems, feel free to file
 # an issue and we can discuss configuration needs.
-set -euo pipefail
 
-if [ "$#" -ne 8 ]; then
-	echo 'create_disk <device> <ostree-repo> <ostree-ref> <ostree-remote> <grub-script> <os-name> <save-var-subdirs> <space separated kargs>'
-	exit 1
-fi
+usage() {
+    cat <<EOC
+${0} create a supermin virtual machinge to create a
+Fedora CoreOS style disk image from an OS Tree.
+
+Options:
+    -R|--ostree-ref: the OSTRee reference to install
+    -d|--disk: disk device to use
+    -g|--grub-script: grub script to install
+    -h|--help: show this helper
+    -k|--kargs: kernel CLI args
+    -o|--osname: the OS name to use, e.g. fedora
+    -r|--ostree-remote: the ostree remote
+    -t|--ostree-repo: location of the ostree repo
+    -v|--save-var-subdirs: "yes" to workaround selabel issue for RHCOS
+
+You probably don't want to run this script by hand. This script is
+run as part of 'coreos-assembler build'.
+EOC
+}
+
+set -exo pipefail
```

Comment:
Can we turn off `-x` for at least arg parsing? I think that'll be pretty verbose and arg parsing isn't usually what I want to see.